unikernel.ml

```diff
@@ -779,6 +779,118 @@ struct
                 ~icon:"/images/robur.png" ())
              `Not_found)
 
+  let edit_policy albatross store uuid reqd (user : User_model.user) =
+    let users = User_model.create_user_uuid_map (snd store).Storage.users in
+    match User_model.find_user_by_key uuid users with
+    | Some u -> (
+        let user_policy =
+          match Albatross.policy albatross ~domain:u.name with
+          | Ok p -> (
+              match p with Some p -> p | None -> Albatross.empty_policy)
+          | Error _ -> Albatross.empty_policy
+        in
+        match Albatross.policy_resource_avalaible albatross with
+        | Ok root_policy ->
```

Comment:
I find the naming here (root_policy) misleading... can we use some other term (it is the unallocated resources)